os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -318,6 +319,19 @@ trwifi_result_e wifi_netmgr_utils_scan_ap(struct netdev *dev, trwifi_ap_config_s
 	return wuret;
 }
 
+trwifi_result_e wifi_netmgr_utils_single_channel_scan(struct netdev *dev, trwifi_scan_config_s *scan_config)

```

Comment:
Who call this function? It looks like that no one uses this function.
And I think you can remove dev from input parameters if dev is not used.

Reply:
Could you merge wifi_netmgr_utils_single_channel_scan to wifi_netmgr_utils_scan_ap???
If config is null, operate the normal scan and if not, operate the single channel scan.
There is no reason to separate them I think.